tests/scripts/release_test.py

```diff
@@ -0,0 +1,42 @@
+from sys import version
+from unittest import TestCase, main
+from pathlib import Path
+from subprocess import Popen, PIPE
+
+p_script_dir = Path(__file__).absolute().parents[0]
+p_root = p_script_dir.parents[1]
+p_VERSION = Path(p_root, "cddl_gen", "VERSION")
+p_RELEASE_NOTES = Path(p_root, "RELEASE_NOTES.md")
+p_HEAD_REF = Path(p_script_dir, "HEAD_REF")
+
+
+class ReleaseTest(TestCase):
+    def test_version_num(self):
+        current_branch = Popen(['git', 'branch', '--show-current'],
```

Comment:
For consideration: the `gitpython` package has means to do this via API commands instead of subprocess calls. It adds a dependency though, so maybe we don't care versus just relying on subprocess.

Reply:
Thanks. I prefer subprocess as long as it's just one or a few lines. To avoid the dependency.